code/modules/research/destructive_analyzer.dm

```diff
@@ -164,3 +126,99 @@ Note: Must be placed within 3 tiles of the R&D Console
 	loaded_item = null
 	update_icon()
 	return TRUE
+
+/obj/machinery/rnd/destructive_analyzer/ui_interact(mob/user)
+	. = ..()
+	var/datum/browser/popup = new(user, "destructive_analyzer", name, 900, 600)
+	popup.set_content(ui_deconstruct())
+	popup.open()
+
+/obj/machinery/rnd/destructive_analyzer/proc/ui_deconstruct()		//Legacy code
+	var/list/l = list()
```

Comment:
Please, use something better than `l`
`dat` or `msg` work if you want something short

Reply:
I can change it, but I wanted to avoid trawling through this dumpster fire of legacy code making style updates, especially when it comes to the Destructive Analyzer which should be fairly trivial to create a tgui interface for which would render this all moot.